packages/webview_flutter_lwe/example/lib/main.dart

```diff
@@ -157,6 +176,13 @@ Page resource error:
         },
       )
       ..loadRequest(Uri.parse('https://flutter.dev'));
+
+    // setBackgroundColor is not currently supported on macOS.
+    if (kIsWeb || !Platform.isMacOS) {
+      controller.setBackgroundColor(const Color(0x80000000));
+    }
+
```

Comment:
This code is not new and does not apply to us, so you can delete it.

Reply:
Because Platform.isTizen is true and this condition affects to our webview, I did not remove it.